controller/src/beerocks/master/db/db.cpp

```diff
@@ -1064,26 +1064,14 @@ bool db::set_station_capabilities(const std::string &client_mac,
     }
 
     if (is_node_5ghz(parent_radio)) {
-        if (n->m_sta_5ghz_capabilities.cap_flag == true) {
-            //skip setting 5Ghz station parameters if it is already learned once
-            LOG(INFO) << "m_sta_5ghz_capabilities already set once";
-        } else {
-            n->m_sta_5ghz_capabilities          = sta_cap;
-            n->m_sta_5ghz_capabilities.cap_flag = true;
-        }
-
-        n->capabilities = n->m_sta_5ghz_capabilities;
+        n->m_sta_5ghz_capabilities          = sta_cap;
+        n->m_sta_5ghz_capabilities.cap_flag = true;
```

Comment:
as this commit headline  implies you can go ahead and remove cap_flag

Reply:
Actually no. When this flag is not set, we return nullptr on capability getter function.

Reply:
So it determined the validity of the data?
In that case, maybe should be changed to `valid`

Reply:
rename it on a separate commit.